tests/fixtures/apply_diff.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+SQL="ATTACH DATABASE @diffDbFilename AS diffDb;
+     DELETE FROM tiles WHERE (zoom_level, tile_column, tile_row) IN (SELECT zoom_level, tile_column, tile_row FROM diffDb.tiles WHERE tile_data ISNULL);
```

Comment:
classical SQL usually uses `IS NULL` (with a space), or `IS NOT NULL`. I am actually surprised sqlite supports the ones without spaces. I'm ok to keep it though if you want

Reply:
`NOTNULL` works as well, I think I will keep it because it's still quite clear what is happening